src/SignalR/clients/csharp/Client.Core/src/Microsoft.AspNetCore.SignalR.Client.Core.csproj

```diff
@@ -32,7 +32,7 @@
     <Reference Include="Microsoft.Bcl.AsyncInterfaces" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(AspNetCoreMajorMinorVersion)' == '3.0'">
+  <ItemGroup Condition="'$(AspNetCoreMajorMinorVersion)' == '5.0'">
```

Comment:
File an issue to update our dependency baselines in this branch to align w/ 3.0 instead of 2.2.

Reply:
Follow ups:

https://github.com/aspnet/AspNetCore/issues/12702
https://github.com/aspnet/AspNetCore-Internal/issues/2893